src/transformation/utils/lua-ast.ts

```diff
@@ -175,11 +176,6 @@ export function createLocalOrExportedOrGlobalDeclaration(
                     declaration = undefined;
                 }
             }
-        } else if (rhs) {
-            // global
-            assignment = lua.createAssignmentStatement(lhs, rhs, tsOriginal);
-        } else {
-            return [];
```

Comment:
Can we really drop that branch? Wouldn't `let foo` in global context be transformed to `local foo` int that case?

Reply:
You are right, reverted